domains/measurements/Measurements.API/Tests/CalculateConsumptionsTest.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using API.Helpers;
+using API.Models;
+using API.Services;
+using Xunit;
+using Xunit.Categories;
+
+namespace Tests;
+
+[UnitTest]
+public sealed class CalculateConsumptionsTest
+{
+    readonly CalculateConsumptionsDataSetFactory dataSetFactory = new();
+
+    [Theory]
+    [InlineData(Aggregation.Total)]
+    [InlineData(Aggregation.Actual)]
+    [InlineData(Aggregation.Hour)]
+    [InlineData(Aggregation.Day)]
+    [InlineData(Aggregation.Month)]
+    [InlineData(Aggregation.Year)]
```

Comment:
Could we have the expected value as inline data instead of using `GetExpectedEmissions`?